lib/Interpreter/DynamicLibraryManager.cpp

```diff
@@ -26,7 +26,11 @@
 #include <Windows.h>
 #include <shlobj.h>
 #else
-#include <limits.h> /* PATH_MAX */
+#ifdef __APPLE__
+ #include <sys/syslimits.h> // PATH_MAX
+#else
+ #include <limits.h> // PATH_MAX
+#endif
```

Comment:
You'd still need PATH_MAX to allocate a buffer to call llvm::sys::fs::make_absolute with.